docs/new/docs/index.md

```diff
@@ -487,52 +490,66 @@ For all the details, see [Every Keyword](./docs/policy-language/#every-keyword).
 Rego lets you encapsulate and re-use logic with rules. Rules are just if-then
 logic statements. Rules can either be "complete" or "partial".
 
-```live:example/complete:module:hidden
-package example.rules
-```
-
 #### Complete Rules
 
 Complete rules are if-then statements that assign a single value to a variable.
 For example:
 
-```live:example/complete/1:module:openable
+```rego
+package rules
+
 any_public_networks := true if {
     some net in input.networks # some network exists and..
     net.public                 # it is public.
 }
 ```
 
+<RunSnippet files="#input.json" command="data.rules.any_public_networks"/>
+
 Every rule consists of a _head_ and a _body_. In Rego we say the rule head
 is true _if_ the rule body is true for some set of variable assignments. In
 the example above `any_public_networks := true` is the head and `some net in input.networks; net.public` is the body.
 
 You can query for the value generated by rules just like any other value:
 
-```live:example/complete/1:query
-any_public_networks
-```
+```rego
+package rules
 
-```live:example/complete/1:output
+any_public_networks := true if {
+    some net in input.networks # some network exists and..
+    net.public                 # it is public.
+}
+
+another_rule := {
+    "public_networks": any_public_networks,
+}
 ```
 
-All values generated by rules can be queried via the global `data` variable.
+<RunSnippet id="public_networks.rego" files="#input.json" command="data.rules.another_rule"/>
 
-```live:example/complete/1/abs:query:merge_down
-data.example.rules.any_public_networks
-```
+All values generated by rules can be queried via the global `data` variable from
+other packages loaded into OPA.
 
-```live:example/complete/1/abs:output:merge_down
+```rego
+package another_package
+
+yet_another_rule := {
+    "public_networks": data.rules.any_public_networks,
+}
 ```
 
-> 💡 You can query the value of any rule loaded into OPA by referring to it with an
-> absolute path. The path of a rule is always:
-> `data.<package-path>.<rule-name>`.
+<RunSnippet files="#input.json #public_networks.rego" command="data.another_package.yet_another_rule"/>
+
+:::info
```

Comment:
We could use `:::tip`, couldn't we? It might be a little less noisy, not too have two symbols right next to each other. Not a blocker!